src/Phinx/Db/Adapter/MysqlAdapter.php

```diff
@@ -749,6 +751,8 @@ public function getSqlType($type, $limit = null)
                 }
 
                 return ['name' => 'blob'];
+            case static::PHINX_TYPE_BIT:
+                return ['name' => 'bit', 'limit' => $limit ? $limit : 64];
```

Comment:
You could do `$limit ?: 64`.

Reply:
Of course, @chinpei215! Thank you!

Reply:
is that available in 5.4?

Reply:
Yes, @dereuromark!
See: http://www.php.net/manual/en/language.operators.comparison.php#language.operators.comparison.ternary